tests/phpunit/src/DkanTestBase.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Dkan\PhpUnit;
+
+use Drupal\Tests\UnitTestCase;
+
+class DkanTestBase extends UnitTestCase {
+
+    protected $dkanDirectory;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setUp() {
+        parent::setUp();
+        $this->dkanDirectory = realpath(dirname(__FILE__) . '/../../../');
+    }
+
+    /**
+     * Helper to call projected methods
+     * 
+     * @param object $object
+     * @param string $methodName
+     * @param variable-lengrh $arguments Additional arguments to pass to 
+     * @return mixed
+     * @throws InvalidArgumentException If method is not defined in object
+     */
+    protected function invokeProtectedMethod($object, string $methodName, ...$arguments) {
```

Comment:
This can be replaced with [`sebastian/peek-and-poke`](https://packagist.org/packages/sebastian/peek-and-poke). It's from the same author as the phpunit packages but he has stopped active development on it and doesn't plan to add features to it.